community/zh-cn/development/developers.md

```diff
@@ -0,0 +1 @@
+## 未完成
```

Comment:
I think it's not necceary to move this file to community.

Reply:
> I think it's not necceary to move this file to community.
The developer currently has no content and has been deleted
![image](https://user-images.githubusercontent.com/28628088/104863606-71c75f00-5971-11eb-83fa-37b1b4f78e59.png)